src/module/mealCounter.js

```diff
@@ -0,0 +1,14 @@
+import { baseUrl } from './apiData.js';
+
+export const mealcounter = async () => {
+  const result = await fetch(baseUrl);
+  const data = await result.json();
+  return data.meals.length;
+};
```

Comment:
For the counter functions, they should look for specific DOM elements and make the counting based on what is actually displayed on the page, not on what the API returns.
Also, kindly add a counter function for the comments, it should be in its own module.